package.json

```diff
@@ -71,7 +71,8 @@
     "utf8": "2.1.1",
     "uuid": "2.0.1",
     "websocket": "^1.0.28",
-    "xhr2-cookies": "^1.1.0"
+    "xhr2-cookies": "^1.1.0",
+    "semver": "6.2.0"
```

Comment:
Does web3.js has the same dependency? I'm not sure, but the dependency of semver should be defined in scrypt.

Reply:
scrypt is included in the web3-eth-accounts package.
scrypt.js uses semver to load libraries with require.
So in package.json inside the web3-eth-accounts package, there is a dependency on semver added. (https://github.com/ethereum/web3.js/commit/d7adf702f815537376e7b8adb8fbdacfece17435#diff-fc046049cbd15ed56a3e1c307b8d70bc)

I missed this part when porting scrypt.

When running locally there was no problem because there was probably dependence on semver in the devDependencies library.
However, it seems to be a problem if use the library install from npm.